src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,23 +6,36 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for i in range(cur_index + 1, len(arr)):
```

Comment:
This works! It's not exactly what the problem asked for, though :) What this does is goes through all of the numbers to the right of current_index and swaps every time a number is smaller than the one at current_index. That means that it's possible to do many swaps in one pass.

The intent of the problem was to have you use the smallest_index (which you didn't use) to keep track of which number is smallest in which position and only swap once.